### PR TITLE
Bhealy21 patch 2

### PR DESCRIPTION
**Pull request by bhealy21**

### CODE REVIEW

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-title: LiveRamp - Distribution connection
-description: Learn how to use the LiveRamp - Distribution connector to activate audiences previously onboarded into LiveRamp, to other advertising destinations.
-hide: yes
-hidefromtoc: yes
+title: LiveRamp - Distribution Connection
+description: Learn how to use the LiveRamp - Distribution connector to orchestrate and activate audiences previously onboarded into LiveRamp, to downstream advertising destinations.
 ---
 
 # [!DNL LiveRamp - Distribution] connection {#liveramp-onboarding}
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 6: Please remove {#liveramp-onboarding}

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 2]
```diff
@@ -11,32 +9,32 @@ The [!DNL LiveRamp - Distribution] connection enables you to activate audiences
...
-* [!DNL Teads]
-* [!DNL WB Discovery]
+* [[!DNL iHeartMedia]](#iheartmedia)
+* [[!DNL Index Exchange]](#index-exchange)
+* [[!DNL Magnite CTV Platform]](#magnite)
+* [[!DNL Magnite DV+ (Rubicon Project)]](#magnite-dv)
+* [[!DNL One Fox]](#fox)
+* [[!DNL Pandora]](#pandora)
+* [[!DNL Reddit]](#reddit)
+* [[[!DNL Roku]](#roku)
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 32: There's an extra `[` on the Roku link

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 3]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
-### [!DNL One Fox] {#fox}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_partnerid"
+>title="PartnerId"
+>abstract="The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use."
 
-### [!DNL Pandora] {#pandora}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_seatid"
+>title="SeatId"
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: >title="SeatId"

The UI field labels should not read like code variables. This should be called "Seat ID" in the customer data fields, labels, and the popover. Please update this to the correct formatting.

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 4]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+>abstract="Magnite DV+ "seatId" provided by your Magnite account manager"
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: >abstract="Magnite DV+ Seat ID provided by your Magnite account manager"

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 5]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite DV+ destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteDV_DestSpecific.png)
+
+* **[!UICONTROL PartnerId]**: The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use.
+* **[!UICONTROL SeatId]**: Magnite DV+ "seatId" provided by your Magnite account manager
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Please update these fields to Partner ID and Seat ID, in both the docs and the destination config.

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 6]
```diff
@@ -110,31 +112,60 @@ To configure details for the destination, fill in the required and optional fiel
...
+>[!CONTEXTUALHELP]
+>id="company_name"
+>title="Your Company Name"
+>abstract="Your Company Name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Ampersand destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Ampersand_DestSpecific.png)
+
+* **[!UICONTROL Your Company Name]**: Your Company Name, as you would like it to be shown to the destination partner. Do not use spaces or special characters.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 132: No need for camelcase in the description: "Your company name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 7]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitectv_client"
+>title="Client"
+>abstract="Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite CTV destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteCTV_DestSpecific.png)
+
+* **[!UICONTROL Client]**: Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 225: No need for capital C in "Client name" description

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 8]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+![Platform UI image showing the customer data fields for the Index Exchange destination.](../../assets/catalog/advertising/liveramp-distribution/LR_IndexExchange_DestSpecific.png)
+
+* **[!UICONTROL Account Name]**: Your Index Exchange client account name. Do not use spaces or special characters.
+
 ### [!DNL Magnite CTV Platform] {#magnite}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitectv_client"
+>title="Client"
+>abstract="Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 219: Lowercase C in the abstract.

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 9]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+![Platform UI image showing the customer data fields for the Magnite CTV destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteCTV_DestSpecific.png)
+
+* **[!UICONTROL Client]**: Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
+
 ### [!DNL Magnite DV+ (Rubicon Project)] {#magnite-dv}
 
-### [!DNL One Fox] {#fox}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_partnerid"
+>title="PartnerId"
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 231: Partner ID

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 10]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_seatid"
+>title="SeatId"
+>abstract="Magnite DV+ "seatId" provided by your Magnite account manager"
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite DV+ destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteDV_DestSpecific.png)
+
+* **[!UICONTROL PartnerId]**: The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 243: Partner ID

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 11]
```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
...
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_fox_client"
+>title="Client"
+>abstract="The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default, but please contact your partner account rep if unsure. No whitespace or other special characters, please (e.g. Company_Name)."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the One Fox destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Fox_DestSpecific.png)
+
+* **[!UICONTROL Client]**: The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default, but please contact your partner account rep if unsure. No whitespace or other special characters, please (e.g. Company_Name).
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 288: [!UICONTROL Client]: The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default. If unsure, please contact your partner account representative. Do not use spaces or special characters (e.g. Company_Name).

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 12]
```diff
@@ -241,22 +358,55 @@ To configure details for the destination, fill in the fields below.
...
+>id="platform_destinations_liveramp_distribution_taboola_seg_type"
+>title="Segment Type"
+>abstract="The segment type (1st Party only available at this time)."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the supported identifiers for the Taboola destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Taboola_DestSpecific.png)
+
+* **[!UICONTROL Account manager email address]**: The email address of your Taboola account manager.
+* **[!UICONTROL Segment Type]**: The segment type (1st Party only available at this time).
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 370: Please update the spelling to "Segment type" in both destination config and docs. Capitalization of fields should be consistent across the UI.

**Review by vfalon — CHANGES_REQUESTED** [review]
@bhealy21 Thanks for the updates. However, it seems some of my comments have not been addressed yet. I see you also created a new PR. This is not needed.

Please review the comments I left in this PR, and _commit the changes to this same PR. (https://github.com/AdobeDocs/experience-platform.en/pull/251/)_ This way we can track the comments and see if there's all resolved.

I will close the previous PR and we'll work on this one.

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 13]
```diff
@@ -110,31 +112,60 @@ To configure details for the destination, fill in the required and optional fiel
...
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Acast destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Acast_DestSpecific.png)
+
+* **[!UICONTROL Client name]**: Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
 
 ### [!DNL Ampersand.tv] {#ampersand-tv}
 
+>[!CONTEXTUALHELP]
+>id="company_name"
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 124: This ID does not follow the correct format. Please update this to:

>id="platform_destinations_liveramp_distribution_ampersand_company_name"
>title="Your company name"
>abstract="Your company name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."

Please update this tooltip ID in the destination config, too. It's using the wrong format:


 "tooltip": {
            "id": "company_name",
            "fallbackUrl": "https://www.adobe.com/go/destinationsliveramp-en"
          }

Also, unrelated to the docs, the **fallbackUrl** in the destination config seems to have been altered. The correct link is  https://www.adobe.com/go/destinations-liveramp-distribution-en

**Review by vfalon — CHANGES_REQUESTED** [review]

**Comment by bhealy21** [comment]
Hey @vfalon .  I believe all changes have now been committed directly to this PR.  Can you please take a look and let me know if this is reflected on your end?

File: help/destinations/catalog/advertising/liveramp-distribution.md (hunk 5 above)
Comment (vfalon) [reply help/destinations/catalog/advertising/liveramp-distribution.md]: @bhealy21 I see you updated this in the destination config, but not in the documentation. Please update these fields to Seat ID and Partner ID.

File: help/destinations/catalog/advertising/liveramp-distribution.md (hunk 10 above)
Comment (vfalon) [reply help/destinations/catalog/advertising/liveramp-distribution.md]: @bhealy21 Please update this to Partner ID.

File: help/destinations/catalog/advertising/liveramp-distribution.md [hunk 14]
```diff
@@ -241,22 +358,55 @@ To configure details for the destination, fill in the fields below.
...
+>id="platform_destinations_liveramp_distribution_taboola_seg_type"
+>title="Segment type"
+>abstract="The Segment type (1st Party only available at this time)."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the supported identifiers for the Taboola destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Taboola_DestSpecific.png)
+
+* **[!UICONTROL Account manager email address]**: The email address of your Taboola account manager.
+* **[!UICONTROL Segment Type]**: The Segment type (1st Party only available at this time).
```
Comment (vfalon) [inline help/destinations/catalog/advertising/liveramp-distribution.md]: Line 370: **[!UICONTROL Segment type]**: The segment type (1st Party only available at this time).

**Review by vfalon — CHANGES_REQUESTED** [review]
Hey @bhealy21 , thanks for the updates. Some of my comments were still not addressed. Please review my latest comments and let me know if you have any questions.

**Pull request merged**